src/azure-cli/requirements.py3.Darwin.txt

```diff
@@ -129,6 +129,7 @@ requests-oauthlib==1.2.0
 requests[socks]==2.31.0
 scp==0.13.2
 semver==2.13.0
+setuptools
```

Comment:
I think items under `requirements.*.txt` should always be pinned as `requirements.txt` is the output of `pip freeze`.

Reply:
We do have a pinned version in `requirements.*.txt`

On the one hand, we always use latest `setuptools` and `pip` when build packages for now. We've used this strategy for years.  https://github.com/Azure/azure-cli/pull/24992#discussion_r1098321100
On the other hand, unpinned `setuptools` caused a problem before: https://github.com/Azure/azure-cli/issues/25869